hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -161,6 +161,152 @@
+                                        <xs:annotation>
+                                            <xs:documentation>
+                                                Includes the ring buffer store factory class name. The store format is the same as
+                                                the in-memory-format for the ringbuffer.
+                                            </xs:documentation>
+                                        </xs:annotation>
+                                        <xs:complexType>
+                                            <xs:attribute name="enabled" use="required" type="xs:boolean"/>
+                                            <xs:attribute name="class-name" use="optional" type="xs:string"/>
+                                            <xs:attribute name="factory-class-name" use="optional" type="xs:string"/>
+                                            <xs:attribute name="store-implementation" use="optional" type="xs:string"/>
+                                            <xs:attribute name="factory-implementation" use="optional" type="xs:string"/>
+                                        </xs:complexType>
+                                    </xs:element>
+                                </xs:sequence>
+                                <xs:attribute name="name" use="required" type="xs:string"/>
+                                <xs:attribute name="capacity" use="optional" type="xs:unsignedInt">
+                                    <xs:annotation>
+                                        <xs:documentation>
+                                            Number of items in the ringbuffer. If no time-to-live-seconds is set, the size will
+                                            always
+                                            be equal to capacity after the head completed the first loop around the ring. This is
+                                            because no items are getting retired.
+                                        </xs:documentation>
+                                    </xs:annotation>
+                                </xs:attribute>
+                                <xs:attribute name="backup-count" use="optional" type="parameterized-backup-count" default="1">
+                                    <xs:annotation>
+                                        <xs:documentation>
+                                            Number of synchronous backups. For example, if 1 is set as the backup-count,
+                                            then the items in the ringbuffer are copied to one other JVM for fail-safety.
+                                            `backup-count` + `async-backup-count` cannot be bigger than maximum
+                                            backup count which is `6`. Valid numbers are 0 (no backup), 1, 2 ... 6.
+                                        </xs:documentation>
```

Comment:
I suppose this doc is copied over javadoc in the config classes but don't know how to keep them in sync? @Serdaro wdyt?